ibm/mas_devops/playbooks/cp4d/db2wh-backup-restore.yml

```diff
@@ -0,0 +1,13 @@
+---
+- hosts: localhost
```

Comment:
It would be better to seperate this into two playbooks ... backup and restore as seperate things ... then in pipelines etc you can either backup only, restore only, or backup followed by a restore.

Reply:
Fixed.